Formula/spdx-sbom-generator.rb

```diff
@@ -0,0 +1,18 @@
+class SpdxSbomGenerator < Formula
+  desc "Support CI generation of SBOMs via golang tooling"
+  homepage "https://github.com/spdx/spdx-sbom-generator"
+  url "https://github.com/spdx/spdx-sbom-generator/archive/refs/tags/v0.0.13.tar.gz"
+  sha256 "7d088f136a53d1f608b1941362c568d78cc6279df9c1bdb3516de075cb7f10c3"
+  license any_of: ["Apache-2.0", "CC-BY-4.0"]
+
+  depends_on "go" => :build
+
+  def install
+    system "make", "build-mac"
+    bin.install "bin/spdx-sbom-generator"
+  end
+
+  test do
+    assert_match /Output Package Manager dependency on SPDX format.*/, shell_output("#{bin}/spdx-sbom-generator --help")
```

Comment:
Hmm. I was thinking about that. The only other thing I can think to do is to download a known git repo and generate an sbom from it but that sounds very resource intensive and we would need to create a repo that will not change

Reply:
Let's try something like that and see if we can simplify it after.

Reply:
Ok so I have created a test that downloads one of my git repos and checks out a specific tag so that we should nt have any changes. Commit: https://github.com/Homebrew/homebrew-core/pull/84924/commits/fb6b7ec5700b18177d323b49da60601d316c9003